Source/UI/RoomsOnboardingView.swift

```diff
@@ -0,0 +1,135 @@
+                            if let range = string.range(of: Localized.Onboarding.yourAliasPlanetary.text) {
+                                string[range].foregroundColor = .highlightGradientAverage
+                            }
+                            if let range = string.range(of: Localized.Onboarding.yourAlias.text) {
+                                string[range].font = .body.italic()
+                            }
+                        }
+                    }
+                    .padding(.bottom, 10)
+                }
+                
+                VStack(spacing: 0) {
+                    ForEach(viewModel.communityAliasServers) { room in
+                        RoomCard(room: room, viewModel: viewModel)
+                            .onTapGesture {
+                                viewModel.selectRoom(room: room)
+                            }
+                    }
+                    
+                    if viewModel.selectedRoom != nil {
+                        Text(Localized.Onboarding.changeAlias.text)
+                            .multilineTextAlignment(.center)
+                            .foregroundColor(.onboardingMainText)
+                            .padding(.top, 30)
+                    }
+                    
+                    Spacer()
+                }
+            }.padding(40)
+            
+            Spacer()
+            
+            VStack(spacing: 20) {
+                
+                Button {
+                    step.next()
+                } label: {
+                    Text(viewModel.selectedRoom == nil ? Localized.Onboarding.aliasServerSkip.text :
+                            Localized.Onboarding.aliasSkip.text).underline()
+                        .foregroundColor(.menuSelectedItemText)
+                        .font(Font(UIFont.systemFont(ofSize: 15, weight: .medium)))
+                }
+                if let room = viewModel.selectedRoom {
+                    Button {
+                        Task {
+                            _ = try await viewModel.joinAndRegister(room: room, alias: viewModel.alias)
+                            self.step.next()
+                            if let identity = step.data.context?.identity {
+                                Onboarding.set(status: .completed, for: identity)
+                            }
+                        }
+                    } label: {
+                        Text(Localized.done.text)
+                            .frame(maxWidth: .infinity)
+                    }
+                    .buttonStyle(PillButtonStyle())
+                    .disabled(!viewModel.aliasIsValid())
+                    .font(Font(UIFont.verse.pillButton))
+                    .frame(maxWidth: .infinity)
+                    .padding([.leading, .trailing], 40)
+                }
+            }.padding(.bottom, 10)
+        }
+    }
+}
+
+struct PillButtonStyle: ButtonStyle {
```

Comment:
Nice! This will be really useful. Can you move it into its own file?

Reply:
Oh, I think this buttonConfiguration is missing a pressed state.

Reply:
>Nice! This will be really useful. Can you move it into its own file?
Done.

>Oh, I think this buttonConfiguration is missing a pressed state.
Added.